addons/binding/org.openhab.binding.seneye/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="seneye"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <thing-type id="seneye">

```

Comment:
1) Renamed the channels ( temperature instead of channeltemperature)
2) It's not just a water (composition) monitor, it also monitors the lightning and the temperature. It could be called aquarium monitor, but I don't think seneye will come up with new products besides their 'seneye monitor'...

Reply:
Well then maybe call the thing-type "monitor"?